### PR TITLE
UTAPI-93: backport ghaction

### PR DESCRIPTION
**Pull request by alexanderchan-scality**

### CODE REVIEW

**Comment by alexanderchan-scality** [comment]
ping

File: lib/UtapiReindex.js [hunk 1]
```diff
@@ -12,6 +12,9 @@ const redisClient = require('../utils/redisClient');
 const REINDEX_SCHEDULE = '0 0 * * Sun';
 const REINDEX_LOCK_KEY = 's3:utapireindex:lock';
 const REINDEX_LOCK_TTL = (60 * 60) * 24;
+const REINDEX_PYTHON_INTERPRETER = process.env.REINDEX_PYTHON_INTERPRETER !== undefined
+    ? process.env.REINDEX_PYTHON_INTERPRETER
+    : 'python3.4';
```
Comment (jonathan-gramain) [inline lib/UtapiReindex.js:17]:
```suggestion
const REINDEX_PYTHON_INTERPRETER = process.env.REINDEX_PYTHON_INTERPRETER || 'python3.4';
```

**Review by jonathan-gramain — APPROVED** [review]
nevermind my suggestion if it's just a dumb backport

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**